seahorn/sea.yaml

```diff
@@ -18,7 +18,7 @@ verify_options:
 # context-sensitive type-aware alias analysis 
   dsa: sea-cs-t
 # weak support for function pointers. sea-dsa is better but less stable
-  devirt-functions: 'sea-dsa'
+  devirt-functions: 'types'
```

Comment:
leave this as `sea-dsa`. That is the correct setting. If needed, set to `types` locally. 

Currently, the warning that is given when `sea-dsa` is used is correct. The lack of warning with `types` is due to over-approximation of targets and should be wrong.

Also, I've noticed other `.DS_Store` files. Please remove all of them.